### DISCUSSION TITLE
How to check?

### DISCUSSION THREAD

**Discussion by dcaccount** [paulmillr/encrypted-dns#73] [Q&A]
Hello,
First of all thanks for your contribution.

I installed the profile. Is there a way to check if DNS encryption is working? I am asking that also because I have noticed a warning next to the WiFi network name:

`this  network is blocking DNS encrypted traffic`

Thanks for helping,
Dan

**Comment by paulmillr** [comment]
https://www.cloudflare.com/ssl/encrypted-sni/

**Comment by dcaccount** [comment]
Thanks, this is the output I got. Is that correct?
It does not seem so. Please advise, thanks[](url)
![FBC49B97-00F4-4CBF-92FB-F01E55745889](https://user-images.githubusercontent.com/38294161/139647628-439edcf0-870f-4b18-8516-b9c83755b626.png)

**Comment by paulmillr** [comment]
If you're using Cloudflare e-dns then it doesn't work. If you're using other provider, it could work. Google for other sites that check for this.
